web_src/js/modules/fomantic/transition.ts

```diff
@@ -8,13 +8,13 @@ export function initFomanticTransition() {
     'set duration', 'save conditions', 'restore conditions',
   ]);
   // stand-in for removed transition module
-  $.fn.transition = function (arg0, arg1, arg2) {
+  $.fn.transition = function (arg0: string | Record<string, any>, arg1: number, arg2: (el: HTMLElement) => void) {
```

Comment:
args could be anything

Reply:
Well, I can type as `any`, it's not like anyone is going to touch these functions.